packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -217,8 +268,13 @@ export abstract class FluidDataStoreContext
 		return this.parentContext.baseLogger;
 	}
 
+	private readonly _deltaManager: ContextDeltaManagerProxy;
```

Comment:
Maybe consider this while keeping the public `get` as `deltaManager`:

```suggestion
	private readonly _contextDeltaManagerProxy: ContextDeltaManagerProxy;
```

I was just surprised to see a call to `this._deltaManager.setReadonly(readonly);` below, which looked very scary based on name alone